sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -150,6 +151,7 @@ public class SystemSchema extends AbstractSchema
       .add("dimensions", ColumnType.STRING)
       .add("metrics", ColumnType.STRING)
       .add("last_compaction_state", ColumnType.STRING)
+      .add("target_num_replicas", ColumnType.LONG)
```

Comment:
Naming suggestions: consider `replication_factor`, which is a standard name. Or simply `replica_count` - avoids the abbreviated form.

Also, what does the prefix "target" in the name denote? Is there a "source"? If there's no ambiguity there, we can drop the prefix.

Reply:
I agree. Renamed to replication_factor.